bundler.go

```diff
@@ -321,16 +348,15 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		return
 	}
 
+	fmt.Println("initial ldflags", b.ldflags.String())
 	// Build ldflags
-	var l = ldflags{
-		"s": []string{},
-		"X": []string{
-			`"main.AppName=` + b.appName + `"`,
-			`"main.BuiltAt=` + time.Now().String() + `"`,
-		},
-	}
+	b.ldflags["s"] = []string{}
+	b.ldflags["X"] = append(b.ldflags["X"],
+		`"main.AppName=`+b.appName+`"`,
+		`"main.BuiltAt=`+time.Now().String()+`"`,
+	)
 	if e.OS == "windows" {
-		l["H"] = []string{"windowsgui"}
+		b.ldflags["H"] = []string{"windowsgui"}
```

Comment:
Same as above, I would rollback to the `l` variable and use the about-to-be created `merge` function of `LDFlags`